js/tests/components/__snapshots__/SnippetPreviewSection.test.js.snap

```diff
@@ -0,0 +1,21 @@
+exports[`SnippetPreviewSection renders the snippet editor inside of it 1`] = `
+<SnippetPreviewSection
+  baseUrl="http://example.org">
+  <Styled(StyledSection)
+    headingIcon="eye"
+    headingIconColor="#555"
+    headingLevel={3}
+    headingText="React snippet preview">
+    <StyledSection
```

Comment:
This is how the snapshot works when using enzyme `mount`. It will show all components a piece of data passes through. The containing `Styled(StyledSection)` receives those arguments and passes them through to its child. That is what you are seeing in the snapshot.

Reply:
Ah, makes sense